src/test/java/com/bookJourney/springboot/mapper/UserMapperTest.java

```diff
@@ -25,4 +27,19 @@ public void testRegistrationRequestDTOtoUser() {
         assertEquals(dto.firstName(), user.getFirstName());
         assertEquals(dto.lastName(), user.getLastName());
     }
+
+    @Test
+    public void testUserToProfileDTO() {
```

Comment:
jeśli nie używasz opisu w postaci @DisplayName staraj się nazywać testy wg konwencji
should<oczekiwany efekt>When<warunek>
ShouldCorrectlyMapUserToProfileDto (część when nie zawsze musi się pojawic ;) )